service/follow_service/follow_action.go

```diff
@@ -0,0 +1,89 @@
+package follow_service
+
+import (
+	"douyin-lite/repository"
+	"errors"
+	"fmt"
+)
+
+const (
+	FOLLOW = iota + 1
+	CANCEL
+)
+
+var (
+	ErrIvdAct    = errors.New("undefined action")
```

Comment:
Maybe we can use a file to store all the custom errors separately.

Reply:
Maybe do this in another pr